bm-instance-agent/bm_instance_agent/manager.py

```
@@ -173,3 +193,368 @@ def console(self):
+                speed = 'UNKNOWN'
+            with open(os.path.join(abspath, 'address'), 'r') as f:
+                mac_address = f.read().strip()
+
+            if len(mac_address) > 32:
+                continue
+            rc, _, _ = bm_utils.shell_cmd("""arping -c 5 -I {} {}""".format(net_dev, provision_net), False)
+            if rc == 0 and not pxe_iface_mac:
+                is_provision_nic = True
+                pxe_iface_mac = mac_address
+
+            is_provision_nic = True if mac_address in pxe_iface_mac else False
```

Ruff (0.11.9)

Remove unnecessary `True if ... else False`

Remove unnecessary `True if ... else False`

(SIM210)